StandingOrders.md

```diff
@@ -0,0 +1,180 @@
+
+### 5.0 Fringe
+
+__5.1__ The Fringe Venue Manager shall decide the number of slots available to EUTC productions prior to the AGM. The Fringe Venue Manager must allocate at least one slot to the EUTC.
+
+__5.2__ A preference system (see clause 8.0 below) shall be used to select EUTC Fringe productions under all circumstances, following the passing of a production merit vote.
+
+__5.3__ The EUTC President, outgoing Fringe Venue Manager and EUSA representatives will interview the Fringe Venue Manager. In the event of conflicts of interest the remaining members of this panel may select a replacement member.
+
+__5.4__ The EUTC President, Fringe Venue Manager and EUSA representatives, together with others as deemed appropriate by this group, will interview for the other three Fringe positions, in the event of a conflict of interest see 5.3.
+
+__5.5__ The EUSA office bearer shall have the right to refer a decision made by the panel to the EUSA Finance Committee only if they feel the interview was not conducted fairly.
+
+### 6.0 Festivals
+
+__6.1__ Any member may propose a Festival to be organised by the EUTC. This must be done at a General Meeting and be submitted one week prior to that meeting. Such a Festival must be budgeted to cover all expenses. Voting will operate as per section 8 below.
+
+__6.2__ The proposal of a Festival must include a named Administrator who is responsible for the running of that Festival and creating a sub-committee of: a Marketing Manager, a Theatre Manager for each venue, a Technical Manager, a Box Office Manager and a Café Manager (if there is to be a box office or café respectively). All persons filling these positions must be members and they must be ratified at a Company Meeting unless they are already part of the festival team by the proposal submission deadline, in which case they shall be considered ratified if the proposal is successful. Any member of the company may propose a motion to ratify a festival team member at a Company Meeting if they have solid reasons to do so.
+
+### 7.0 Procedure
+
+__7.1__ The proposers of all shows and all persons standing for election have the right to a short speech (duration to be specified by the Chair), followed by questions as directed by the Chair.
+
+### 8.0 Voting
+
+A Single Transferable Vote, or STV, shall be used in all cases where a preference vote is required. The voting and counting procedure is detailed below. For simplicity, ‘candidate’ shall refer to any person, show or proposal standing in an election. In any preference vote a decision against all candidates is valid and shall be indicated by the words ‘none’ or ‘no-one’, as appropriate. In any vote where only one candidate stands, a STV may be performed between the candidate and ‘none’ or ‘no-one’. This is equivalent to a simple YES/NO/ABSTAIN majority vote, where ABSTAIN is neither a YES, nor a NO, and will only be counted towards ensuring that quorum is met.
```

Comment:
Why is this duplicated?

Reply:
shouldn't be ill re-locate original wording

Reply:
actually what is duplicated?

Reply:
This, and the paragraph next in the document shares a lot of wording. May just be a typo in the document however.

Reply:
yeah although they are the same and this seems to be what the original says (aside from last line of 8.0.1 which is from an amendment

Reply:
Fair enough